### PR TITLE
Bug Fixes 0.0.5rc

### PR DESCRIPTION
**Pull request by sethdenner**
Cleaned up some bugs that were uncovered while working with a test project. I'll modify the commit message to reflect the changes made.

### CODE REVIEW

File: djangocassandra/db/backends/cassandra/compiler.py [hunk 1]
```diff
@@ -189,7 +210,8 @@ def get_row_range(self, range_predicates):
         return self._get_rows_by_indexed_column(range_predicates)
     
     def get_all_rows(self):
-         return self.cql_query.all()
+        import pdb; pdb.set_trace()
```
Comment (simlay) [inline djangocassandra/db/backends/cassandra/compiler.py]: This probably shouldn't be here.

File: djangocassandra/db/backends/cassandra/compiler.py [hunk 2]
```diff
@@ -220,12 +242,14 @@ def count(
         self,
         limit=None
     ):
+        import pdb; pdb.set_trace()
```
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/compiler.py]: don't forget to remove.

File: djangocassandra/db/backends/cassandra/base.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 
 
 class DatabaseFeatures(NonrelDatabaseFeatures):
+
```
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/base.py]: why this new line?

File: djangocassandra/db/backends/cassandra/compiler.py [hunk 4]
```diff
@@ -125,12 +127,18 @@ def _get_rows_by_indexed_column(self, range_predicates):
                     predicates_by_column[column]
                 )
 
-        assert len(range_predicates) == len(sorted_predicates)
```
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/compiler.py]: should replace this assert, removed for testing.

File: djangocassandra/db/backends/cassandra/compiler.py (hunk 1 above)
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/compiler.py]: don't forget to remove this either.

File: djangocassandra/db/backends/cassandra/compiler.py [hunk 5]
```diff
@@ -220,12 +242,14 @@ def count(
...
         limit=None
     ):
+        import pdb; pdb.set_trace()
         return self.cql_query.count()
 
     def delete(
         self,
         columns=set()
     ):
+        import pdb; pdb.set_trace()
```
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/compiler.py]: more pdb

File: djangocassandra/db/backends/cassandra/schema.py [hunk 6]
```diff
@@ -93,7 +93,13 @@ def alter_db_table(
...
             7) clean up OLD_Keyspace (drop schema, delete files, etc.)
         '''
-        pass
+        if (
+            new_db_table._meta.db_table !=
+            old_db_table._meta.db_table
+        ):
+            raise Exception('Renaming models not supported')
+
+        self.create_model(new_db_table)
```
Comment (sethdenner) [inline djangocassandra/db/backends/cassandra/schema.py]: Oh yeah, I also added support for the alter table which is used a lot in the migration scipts. This is the same as creating a model in cqlengine.

File: djangocassandra/db/models.py [hunk 7]
```diff
@@ -5,6 +5,8 @@
 )
 from cassandra.cqlengine.management import create_keyspace
 
+from .fields import AutoFieldUUID
```
Comment (sethdenner) [inline djangocassandra/db/models.py]: unnecessary import?

File: djangocassandra/db/models.py [hunk 8]
```diff
@@ -82,6 +84,7 @@ def __new__(
             '''
             primary_key_field = model._meta.pk
             column_type = get_cql_column_type(primary_key_field)
+
```
Comment (sethdenner) [inline djangocassandra/db/models.py]: really mean to add newline here?

**Pull request closed without merging**